andela/ah-kifaru-frontend
# #167640442 Setup testing with Jest and Enzyme

#### What does this PR do?
Setup testing with Jest and Enzyme

#### Description of Task to be completed?
```
- ensure Jest and Enzyme are setup for testing
- write tests to ensure setup is working
```

#### How should this be manually tested?
The following steps can be used:
1) Clone the repository 
2) Cd into the project folder 
3) Run `npm install` to install all project dependencies. 
4) On the terminal run `npm test`
5) There should be a response showing that tests are passing

#### Any background context you want to provide?
- Ensure that node and npm are installed on your machine

#### What are the relevant pivotal tracker stories?(if applicable)
chore #167640442: https://www.pivotaltracker.com/story/show/167640442

#### Screenshots
![image](https://user-images.githubusercontent.com/31288326/62280449-d7edb880-b443-11e9-8079-3cc3434efc6e.png)
